slurm_submit

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#! sbatch directives begin here ###############################
+#! Name of the job:
+#SBATCH -J covid-sim-model
+#! Charge the SCRC project:
+#SBATCH -A DIRAC-DC003-CPU
+
+#! Set file paths for output to stdout and stderr:
+#! %A means slurm job ID and %a means array index
+#! (Note: It appears that these can _not_ go in the folders we create for each job)
+#SBATCH --output=job_%A_%a_std.out
+#SBATCH --error=job_%A_%a_std.err
+
+#! Simple single core jobs use one node and one task
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#! Set a time limit of 5 minutes
```

Comment:
Possibly need a TODO so the user knows to change the wallclock time if they are using particularly large parameters.